src/services/loginService.js

```diff
@@ -0,0 +1,26 @@
+const RegistrationService = require('./registrationService');
+
+class LoginService {
+  constructor(conn) {
+    this.conn = conn;
+  }
+
+  async authorizeUser(input) {
+    const registrationService = new RegistrationService(this.conn);

```

Comment:
this should be passed through constructor parameter as a dependency injection. you shouldn't create a new service instance every time even if it's not a problem in this case. this comment is referred from routes/index.js

Reply:
corrected